### PR TITLE
ISSUE-12 Mikhail Ivanov

### PR DESCRIPTION
**Pull request by ivanovmp**
I hadn't found a way to rename a branch and preserve its presence in pull request, so the only option left was to create a new pull request from the very beginning.

### CODE REVIEW

**Comment by ivanovmp** [comment]
Maybe, this time I did everything fine? Are the tasks solved well?

File: tasks [hunk 1]
```diff
@@ -17,7 +17,7 @@
 39 - 49
 54 - 76
 72 - 38
-97 + 71
```
Comment (cczy1910) [inline tasks]: Neh, still having some problems. Try to solve it again.

**Comment by ivanovmp** [comment]
Am I lucky this time?

**Comment by ivanovmp** [comment]
By the way, the ticket we are dealing with is here: [link](https://github.com/cscenter/practical-minimum-2019-students/issues/12).

**Pull request merged**